src/main/java/cinema/Main.java

```diff
@@ -15,26 +15,27 @@
 import cinema.service.ShoppingCartService;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import org.apache.log4j.Logger;
 
 public class Main {
+    private static final Logger LOGGER = Logger.getLogger(Main.class);
     private static Injector injector = Injector.getInstance("cinema");
 
     public static void main(String[] args) {
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
-        movieService.getAll().forEach(System.out::println);
         Movie movie = new Movie();
         movie.setTitle("Fast and Furious");
         movie.setDescription("I never heard of the film");
         movieService.add(movie);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(LOGGER::debug);
```

Comment:
Better use info, not debug

Reply:
why?

Reply:
with debug level we log in those information which we want to see if something has happened ( some object state from which depend the program running e.g)